packages/breadboard/README.md

```diff
@@ -1,156 +1,696 @@
+
+## Installation
+
+Breadboard requires [Node.js](https://nodejs.org/) version 19 or higher. Before installing, [download and install Node.js](https://nodejs.org/en/download/).
+
+- Check what version of Node.js you're running with `node -v`.
+- In your workspace, make sure to create a `package.json` first with the [`npm init` command](https://docs.npmjs.com/creating-a-package-json-file).
+
+To install Breadboard with [npm](https://www.npmjs.com/), run:
+
+```shell
+npm install @google-labs/breadboard
+```
 
-:two: **Modularity and composability**. Easily share, remix, reuse, and compose prototypes.
+If you want to use [TypeScript](https://www.typescriptlang.org/), you will need to [install the package](https://www.npmjs.com/package/typescript).
 
-## Requirements
+- You can use npm to download TypeScript into your project using `npm install typescript --save-dev`.
+- You can then initialize the TypeScript project and create a `tsconfig.json` file using `npx tsc --init`.
 
-Breadboard requires [Node](https://nodejs.org/) version >=v19.0.0.
+> **For Windows Users**: For compatability, [Dev Containers](https://code.visualstudio.com/docs/devcontainers/containers) are recommended for ease of use.
 
-> [!WARNING]
-> The library is in active development, and we're transitioning to new syntax. Stuff below will likely be out of date.
-> Please bear with us as we bring our words up to speed with our thoughts/actions.
+### Useful packages (optional)
 
-## Installing the library
+| Name                                                                                            | Description                                                                                                                                    | Install                                   |
+| ----------------------------------------------------------------------------------------------- | ---------------------------------------------------------------------------------------------------------------------------------------------- | ----------------------------------------- |
+| [Breadboard Core Kit](https://www.npmjs.com/package/@google-labs/core-kit)                      | Breadboard kit for foundational board operations like `map` and `invoke`. This contains operations that enable composition and reuse of boards | `npm install @google-labs/core-kit`       |
+| [Breadboard CLI](https://github.com/breadboard-ai/breadboard/tree/main/packages/breadboard-cli) | Command-line tool for generating, running, and debugging boards. This lets you run and build breadboards directly from the command-line.       | `npm install @google-labs/breadboard-cli` |
 
-To install breadboard, run:
+You can find many other helpful [Breadboard packages](https://github.com/breadboard-ai/breadboard/blob/main/README.md#packages) available.
 
-```sh
-npm install @google-labs/breadboard
+## Usage
+
+### Making your first board
+
+```typescript
+import { board } from "@google-labs/breadboard";
```

Comment:
We'll need to convert this to the build API syntax now 😭 
https://github.com/breadboard-ai/breadboard/tree/main/packages/build

Reply:
Absolutely! Shall we add a note near the top to say that it's WIP and shall be migrated to the new build API syntax?

Reply:
Ok!